packages/dev/core/src/FlowGraph/Blocks/Execution/Animation/flowGraphPlayAnimationBlock.ts

```diff
@@ -1,24 +1,30 @@
 import type { FlowGraphContext } from "../../../flowGraphContext";
-import type { Animatable, Animation, IAnimatable } from "../../../../Animations";
+import type { Animatable, Animation } from "../../../../Animations";
 import type { FlowGraphDataConnection } from "../../../flowGraphDataConnection";
 import type { FlowGraphSignalConnection } from "../../../flowGraphSignalConnection";
 import { FlowGraphAsyncExecutionBlock } from "../../../flowGraphAsyncExecutionBlock";
 import { RichTypeAny, RichTypeNumber, RichTypeBoolean } from "../../../flowGraphRichTypes";
 import { RegisterClass } from "../../../../Misc/typeStore";
 import type { IFlowGraphBlockConfiguration } from "../../../flowGraphBlock";
+import type { FlowGraphPath } from "../../../flowGraphPath";
+
+export interface IFlowGraphPlayAnimationBlockConfiguration extends IFlowGraphBlockConfiguration {
+    targetPath: FlowGraphPath;
+    animationPath: FlowGraphPath;
+}
 /**
  * @experimental
  * A block that plays an animation on an animatable object.
  */
 export class FlowGraphPlayAnimationBlock extends FlowGraphAsyncExecutionBlock {
     /**
-     * Input connection: The target to play the animation on.
+     * Input connection: The possible template strings to substitute for in the target path
      */
-    public readonly target: FlowGraphDataConnection<IAnimatable>;
+    public readonly templateStringTargetInputs: FlowGraphDataConnection<number>[] = [];

```

Comment:
This means that in order to define a specific target, I will need to either know its direct path, or the ID I can access it with. Will the developer know this information? Or will this information be provided by the context?

Reply:
I believe in this case the developer needs to know the ID to access the path, since if they create a path with `a/{b}/c`, then they will know there is an input named `b` in the block.